usnistgov/oar-midas-portal
# Use authentication token when pulling table data from dbio

PR #43 completed the transition for using the common authentication service to pull an authentication token into the portal application.  This PR ensures that the token is then used when fetching data from the dbio.  Currently, the token is pulled in via the `LandingComponent`, but other child components are responsible for actually pulling data from the dbio service.  In this PR, the token is passed from the former to the child components that need it using the `@Input()` mechanism. 

This PR ensures that the token is used when pulling DAP and DMP records.  Use of a token as not yet been applied in the following of cases:
   * populating the modal tables
   * populating the file spaces table

To test, 
  * run under oar-docker (branch [test/m1-build](https://github.com/usnistgov/oar-docker/tree/test/m1-build))
  * view the portal home page
  * ensure that the records listed correspond to the logged in user.
